### DISCUSSION TITLE
训练中途可以修改学习率吗？

### DISCUSSION THREAD

**Discussion by TQG1997** [svc-develop-team/so-vits-svc#360] [Q&A]
可不可以在刚开始训练时用较大的学习率（0.0004），中途再改为正常的（0.0002,0.0001）

**Comment by PriesiaMioShirakana** [comment]
可以，但是不建议

**Comment by PriesiaMioShirakana** [comment]
学习率会自动衰减

**Comment by fish5257** [comment]
想问问怎么中途改学习率

**Comment by TQG1997** [comment]
> 想问问怎么中途改学习率

修改配置文件

**Comment by fish5257** [comment]
> > 想问问怎么中途改学习率
> 
> 修改配置文件

额 这样改不成吧 学习率已经存在.pth中了 config中的learning_rate是初始的学习率 后面训练中的log会输出lr: 3.831874885028449e-05  训练中的学习率已经不从config读了吧 lr_decay或许会读

**Comment by 2575156125** [comment]
可以自己改文件，在每个epoch中检查设置文件是否有改动，如果改动了，就重新加载优化器
